bc-backend/src/main/java/org/jboss/da/bc/impl/ProjectHiearchyCreator.java

```diff
@@ -112,6 +111,7 @@ private void setDependencies(ProjectHiearchy hiearchy) {
             hiearchy.setAnalysisStatus(DependencyAnalysisStatus.ANALYZED);
         } catch (CommunicationException ex) {
             log.warn("Failed to get dependencies for " + gav, ex);
+            hiearchy.getProject().addError(BcError.NO_DEPENDENCY);
             hiearchy.setAnalysisStatus(DependencyAnalysisStatus.FAILED);
         } catch (FindGAVDependencyException ex) {
             ProjectDetail project = toplevel.getProject();
```

Comment:
You're right, but probably we should add it in the pom and scm exception like we discussed on irc